code/datums/gamemode/role/role.dm

```diff
@@ -229,7 +230,17 @@
 	return
 
 /datum/role/proc/process()
-	return
+	var/mob/M = antag.current
+	if(!destroyed)
+		if(!M)
```

Comment:
Shouldn't it be !M || M.gcDestroyed? I assume you're looking for if the mob no longer exists

Reply:
We actually have a new role-level destroyed var now. (see L97)

Reply:
That doesn't answer my question. Are you looking to see if the owner of this role was destroyed or not?

Reply:
That's right, although the owner mob can change (e.g.: person gets debrained, they are now inside the brain mob)

Although it is near inconceivable that somehow the mob could be destroyed and later the role owner might have a mob again, I wanted to have sanity for that